vm/devices/net/net_mana/src/lib.rs

```diff
@@ -1397,10 +1539,11 @@ mod tests {
             assert_eq!(packets_n, 1);
             let rx_id = packets[0];
 
-            let mut received_data = vec![0; PACKET_LEN];
+            let mut received_data = vec![0; packet_len];
             payload_mem
                 .read_at(2048 * rx_id.0 as u64, &mut received_data)
                 .unwrap();
+            assert!(received_data.len() == packet_len);
```

Comment:
nit: `assert_eq!` will give you better output on failure, to make debugging easier